### PR TITLE
Feature/webuy

### PR DESCRIPTION
**Pull request by buhills**

### CODE REVIEW

File: assets/styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  width: 25rem;
+}
+
+.footer-container-logo-png {
+  height: 4vh;
+  display: inline-block;
+  padding-top: 15px;
+}
+
+.footer-container-logo-txt {
```
Comment (nganifaith) [inline assets/styles/styles.css:181]: provide some margins to your section

**Review by nganifaith — CHANGES_REQUESTED** [review]
Great job so far @buhills.  I love the work you've put in. However, we are not yet there.

**Changes requested**

- [x] your font size is too big
- [x] you page is leaking
- [x] avoid using fixed margins and padding as much as possible. Use relative values like auto, %, em, rem, vw, vh.
- [x] complete your footer
- [x] center you text in every container.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,105 @@
...
+    <link rel="stylesheet" href="./assets/styles/styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  </head>
+  <body>
+    <header class="headercontainer">
+      <div class="headercontainer-header">
+        <div class="material-symbols-outlined">
+          menu
+        </div>
+        <div class="webuy">
```
[the remark this answers is not in the document]
Comment (buhills) [reply index.html]: copied

File: index.html [hunk 3]
```diff
@@ -1,12 +1,105 @@
...
+    <title>we-buy</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" />
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./assets/styles/styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  </head>
+  <body>
+    <header class="headercontainer">
+      <div class="headercontainer-header">
```
[the remark this answers is not in the document]
Comment (buhills) [reply index.html]: copied

File: index.html [hunk 4]
```diff
@@ -1,12 +1,105 @@
...
+          <div>
+            <button class="contactus-container-form-button"><span class="material-symbols-outlined">call</span> CALL
+              US</button><br><br>
+            <span>Unsure about your car's valuation</span><br><br>
+            <button class="contactus-container-form-button">READ FAQ</button>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="footer-container">
```
[the remark this answers is not in the document]
Comment (buhills) [reply index.html:159]: Done

**Review by nganifaith — COMMENTED** [review]
Great job @buhills. still have some work to do.

- [ ] increase the font size of your footer element
- [ ] lastest news section reduce the gap and increase the card size
- [ ] header out of place
- [ ] why sell with us section is out of place. Font size shouldn't be bold

**Comment by buhills** [comment]
> Great job @buhills. still have some work to do.
> 
> - [x] increase the font size of your footer element
> - [ ] lastest news section reduce the gap and increase the card size
> - [ ] header out of place
> - [ ] why sell with us section is out of place. Font size shouldn't be bold

Copied ma'am 
Thanks

**Pull request merged**